ebpf/event_common.go

```diff
@@ -87,3 +92,25 @@ func (c ConnectionStats) ByteKey(buffer *bytes.Buffer) ([]byte, error) {
 	}
 	return buffer.Bytes(), nil
 }
+
+func removeDuplicates(conns []ConnectionStats) []ConnectionStats {
+	connections := make([]ConnectionStats, 0, len(conns))
+	seen := map[string]struct{}{}
+	buf := &bytes.Buffer{}
```

Comment:
we seem to be mixing slice and map initialization styles in this same code base. It'd be nice to just stick with one style (I think `make(...)` seems to be used elsewhere)